rmgpy/kinetics/falloff.pyx

```diff
@@ -228,9 +230,19 @@ cdef class Lindemann(PDepKineticsModel):
         assert isinstance(ct_reaction, ct.FalloffReaction), "Must be a Cantera FalloffReaction object"
 
         ct_reaction.efficiencies = PDepKineticsModel.get_cantera_efficiencies(self, species_list)
-        ct_reaction.high_rate = self.arrheniusHigh.to_cantera_kinetics()
-        ct_reaction.low_rate = self.arrheniusLow.to_cantera_kinetics()
-        ct_reaction.falloff = ct.Falloff()
+        ct_reaction.rate = self.to_cantera_kinetics() 
+        
+
+    def to_cantera_kinetics(self): 
+        """
+        Converts the Lindemann object to a cantera LindemannRate object
+        """
+        import cantera as ct
+
+        high_rate = self.arrheniusHigh.to_cantera_kinetics(ArrObj=True)
+        low_rate = self.arrheniusLow.to_cantera_kinetics(ArrObj=True)
+        return ct.LindemannRate(low_rate, high_rate, [])
```

Comment:
I'm putting this here so the question is answered, because the update I'm making does not pass a third argument. In Cantera, `LindemannRate`  can accept an additional falloff parameterization:
<img width="327" alt="image" src="https://user-images.githubusercontent.com/56306881/236589130-49332cbc-93e3-4f6e-9b41-058f1f8bf78e.png">

But if nothing is specified, it uses the same form rmg uses, shown below: 
<img width="270" alt="image" src="https://user-images.githubusercontent.com/56306881/236589159-8a5cff18-3a94-4d8b-8bda-0a094e0a3cfc.png">